### ISSUE TITLE
taxon and/or interaction type and/or spatial selection should constrain map display

### ISSUE THREAD

**Issue by jhammock** [jhpoelen/eol-globi-data#168]
This is probably GloBI III, and may be documented already; feel free to close as duplicate if needed. 

If some selection has been made in the top left of the browser, and/or the area picker has been used, the points displayed on the map should be filtered by that query.

**Comment by jhpoelen** [comment]
I totally agree that this would help make the exploration process a bit more intuitive. This might take a little work, so I'll assign it to the GloBI III bucket. If it is super-urgent, please let me know.

**Comment by jhammock** [comment]
sounds good!
